common/src/main/java/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -22,6 +22,7 @@ class AppConfig(
     val baseURL: URL = proxyURL?.also {
         log(LogIntent.INFO, ConfigureStrings.CONFIGURING_PURCHASES_PROXY_URL_SET)
     } ?: URL("https://api.revenuecat.com/")
+    val dianosticsURL = URL("https://api-telemetry.revenuecat.com/") // WIP: Update URL
```

Comment:
I don't think we need to allow a proxy for this. I added a WIP since it's likely the name of this subdomain will change.